docs/source/whatsnew/1.3.0.txt

```diff
@@ -0,0 +1,90 @@
+Development
+-----------
+
+:Release: 1.3.0
+:Date: TBD
+
+Highlights
+~~~~~~~~~~
+
+Support for Newer Numpy/Pandas Versions
+```````````````````````````````````````
+
+Zipline has historically been very conservative when updating versions of
+numpy, pandas, and other "PyData" ecosystem packages. This conservatism is
+primarily due to the fact that Zipline is used as the backtesting engine for
+[Quantopian](https://www.quantopian.com/), which means that updating package
+versions risks breaking a large installed codebase. Of course, many Zipline
+users don't have the backwards compatibility requirements that Quantopian has,
+and they'd like to be able to use the latest and greatest package versions.
+
+As part of this release, we're now building and testing Zipline with two
```

Comment:
Should we also link the PR for this (#2194) here? On the chance that people don't read the individual contributions